relay-general/src/store/normalize.rs

```diff
@@ -254,6 +255,33 @@ impl<'a> NormalizeProcessor<'a> {
     }
 }
 
+fn normalize_app_start_spans(event: &mut Event) {
```

Comment:
As a small thing it would be great to add docs for the newly added function.

Reply:
Thanks, added.